src/manage_sql_secinfo.c

```diff
@@ -171,6 +171,95 @@ increment_transaction_size (int* current_size)
     }
 }
 
+/**
+ * @brief Split a file.
+ *
+ * @param[in]  path  Path to file.
+ * @param[in]  size  Approx size of split files.
+ *
+ * @return Temp dir holding split files.
+ */
+static const gchar *
+split_xml_file (gchar *path, const gchar *size)
+{
+  int ret;
+  static gchar dir[] = "/tmp/gvmd-split-xml-file-XXXXXX";
+  gchar *previous_dir, *command;
+
+  if (mkdtemp (dir) == NULL)
+    {
+      g_warning ("%s: Failed to make temp dir: %s",
+                 __func__,
+                 strerror (errno));
+      return NULL;
+    }
+
+  previous_dir = getcwd (NULL, 0);
+  if (previous_dir == NULL)
+    {
+      g_warning ("%s: Failed to getcwd: %s",
+                 __func__,
+                 strerror (errno));
+      return NULL;
+    }
+
+  if (chdir (dir))
+    {
+      g_warning ("%s: Failed to chdir: %s",
+                 __func__,
+                 strerror (errno));
+      g_free (previous_dir);
+      return NULL;
+    }
+
+  if (gvm_file_copy (path, "split.xml") == FALSE)
+    {
+      g_free (previous_dir);
+      return NULL;
+    }
+
```

Comment:
It would be nice to have a short description of the different files produced in the split command:
What do the intial `split-*.xml` files contain? What should the head and tail consist of?